catalog/spatial/registry/registry-rest-endpoint/pom.xml

```diff
@@ -50,11 +50,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <version>3.4</version>
-        </dependency>
-        <dependency>
-            <groupId>org.codice.ddf.registry</groupId>
-            <artifactId>registry-schema-bindings</artifactId>
+            <version>3.0</version>
```

Comment:
@lessarderic that is a great question. Although both dependency management and the version property work, it seems like dependency management is more appropriate and designed for multi-module cases. However, I like @coyotesqrl idea of bundle wrapping this one or going with the bundle 0 approach.